src/afl-fuzz-bitmap.c

```diff
@@ -522,6 +522,20 @@ save_if_interesting(afl_state_t *afl, void *mem, u32 len, u8 fault) {
 
     if (afl->queued_paths % 10 == 0) {
 
+      u64 singleton = 0;
+      for (u32 i = 0; i < afl->queued_paths; i++) {
+
+        struct queue_entry *q = afl->queue_buf[i];
+        if (afl->n_fuzz_tmp[q->n_fuzz_entry] == 1) { ++singleton; }
+
+      }
+
+      if (singleton) {
+        afl->time_to_next = (afl->prev_run_time + get_cur_time() - afl->start_time) / (double) singleton;

```

Comment:
What's the purpose of `afl->prev_run_time`?

I would assume that `get_cur_time() - afl->start_time` already gives you the total duration of the campaign.

Reply:
Not sure. That's just how campaign length was computed elsewhere.

Reply:
I see... :D Maybe it's the running time from a previous campaign that's loaded?!?